addons/event/data/event_data.xml

```diff
@@ -9,7 +9,6 @@
         </record>
         <record id="event_stage_booked" model="event.stage">
```

Comment:
As those records are in noupdate, you probably need a migration script with a "if_unchanged / update_record_from_xml" combo.

Reply:
Hello @awa-odoo , 
> if_unchanged / update_record_from_xml

seems to working fine with `event_stage_done`, but for `event_stage_booked` it doesn't update the record !!
Can we do it like  `<field name="description"></field>` ?
(btw now I will take care of this task)

Reply:
Hello @mrsr-odoo 

That's interesting, I never considered that before.
Then yes we can specify a blank field value instead, and add a code comment mentioning it's necessary to wipe the previous value during the upgrade.